### PR TITLE
Fix a link

### PR DESCRIPTION
**Pull request by marcjansen**
Title says it all.

### CODE REVIEW

File: jekyll/_cci2/env-vars.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ When the above config runs, the output looks like this:
 
 ![Env Vars Interpolation Example]({{site.baseurl}}/assets/img/docs/env-vars-interpolation-example.png)
 
-You may have noticed that there are two similar steps in the above image and config - "What branch am I on?". These steps illustrate two different methods to read environment variables. Note that both `${VAR}` and `$VAR` syntaxes are supported. You can read more about shell paramater expansion in the [Bash documentation (https://www.gnu.org/software/bash/manual/bashref.html#Shell-Parameter-Expansion).
+You may have noticed that there are two similar steps in the above image and config - "What branch am I on?". These steps illustrate two different methods to read environment variables. Note that both `${VAR}` and `$VAR` syntaxes are supported. You can read more about shell paramater expansion in the [Bash documentation] (https://www.gnu.org/software/bash/manual/bashref.html#Shell-Parameter-Expansion).
```
Comment (michelle-luna) [inline jekyll/_cci2/env-vars.md]:
```suggestion
You may have noticed that there are two similar steps in the above image and config - "What branch am I on?". These steps illustrate two different methods to read environment variables. Note that both `${VAR}` and `$VAR` syntaxes are supported. You can read more about shell paramater expansion in the [Bash documentation] (https://www.gnu.org/software/bash/manual/bashref.html#Shell-Parameter-Expansion).
```
```suggestion
You may have noticed that there are two similar steps in the above image and config - "What branch am I on?". These steps illustrate two different methods to read environment variables. Note that both `${VAR}` and `$VAR` syntaxes are supported. You can read more about shell paramater expansion in the [Bash documentation] (https://www.gnu.org/software/bash/manual/bashref.html#Shell-Parameter-Expansion).
```

Comment (michelle-luna) [inline jekyll/_cci2/env-vars.md]:
```suggestion
You may have noticed that there are two similar steps in the above image and config - "What branch am I on?". These steps illustrate two different methods to read environment variables. Note that both `${VAR}` and `$VAR` syntaxes are supported. You can read more about shell paramater expansion in the [Bash documentation](https://www.gnu.org/software/bash/manual/bashref.html#Shell-Parameter-Expansion).
```

**Comment by michelle-luna** [comment]
can you click the commit suggestion button? I can't commit the change myself. thank you!

**Comment by marcjansen** [comment]
Thanks for the review, @michelle-luna, updated the PR.

**Pull request merged**